Calculator7/Calculator2.iml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not needed anymore as IDEA can import maven projects.
It will take a one-time effort to setup the IDEA project from maven though. Local IDEA project files need not to be added to the source control.

Maven's `pom.xml` file should be treated as single source of truth, that is any relevant project change should be done in the `pom.xml` and then IDEA should pick it up.